### PR TITLE
Minor suggestions

### PR DESCRIPTION
**Pull request by imiric**
Don't think we need to duplicate the removal instructions, let's just link to the installation page.

### CODE REVIEW

File: debian/DEBIAN/control [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: k6
-Version: 0.32.1-2.deprecaterepo
+Version: 0.31.1-2.deprecaterepo
```
Comment (mstoykov) [inline debian/DEBIAN/control:2]: :facepalm:

File: debian/DEBIAN/preinst [hunk 2]
```diff
@@ -1,9 +1,6 @@
 #!/bin/sh -e
 
-echo "\e[32mThe repo you are using for installing k6 is being depracated as a result bintray sunsetting it's service on 1st of May 2021.\e[0m
-As a result of that there is now a new repository. In order to upgrade we recommend that you:
-1. removing this repo which if you've followed the old installation instructions should be done by deleting the line 'deb https://dl.bintray.com/loadimpact/deb stable main' from /etc/apt/sources.list
-2. install the new gpg key 'sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69' as said in https://k6.io/docs/getting-started/installation/#fedora-centos
-3. install the new repo 'echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list' as said in https://k6.io/docs/getting-started/installation/#fedora-centos
-4(optional). reinstall the package with 'sudo apt-get update && sudo apt-get reinstall k6'"
+echo "\e[32mThe repo you are using for installing k6 is being shutdown as a result of Bintray sunsetting its service on May 1st 2021.\e[0m
```
Comment (na--) [inline debian/DEBIAN/preinst]:
```suggestion
echo "\e[32mThe repo you are using for installing k6 is being shut down as a result of Bintray sunsetting its service on May 1st 2021.\e[0m
```

File: debian/DEBIAN/preinst [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
+The k6 team has made available a new repository where future packages will be published.
```
Comment (na--) [inline debian/DEBIAN/preinst]:
```suggestion
The k6 team has created a new package repository where future packages will be published.
```

File: k6.spec [hunk 4]
```diff
@@ -9,17 +9,16 @@ Summary: Load testing for the 21st century
 k6 is a developer centric open source load testing tool for testing the performance of your backend infrastructure. It’s built with Go and JavaScript to integrate well into your development workflow, so you can stay on top of performance without fuzz.
 
 %pre -p /bin/sh
-echo -e "\e[32mThe repo you are using for installing k6 is being depracated as a result bintray sunsetting it's service on 1st of May 2021.\e[0m
-As a result of that there is now a new repository. In order to upgrade we recommend that you:
-1. removing this repo which if you've followed the old installation instructions should be done with 'sudo rm /etc/yum.repos.d/bintray-loadimpact-rpm.repo' if not you shoudl just delete the repo file you created when installing
-2. install the new repo 'sudo dnf install https://dl.k6.io/rpm/repo.rpm' as said in https://k6.io/docs/getting-started/installation/#fedora-centos
-3(optional). reinstall the package with 'dnf reinstall k6'"
+
+echo -e "\e[32mThe repo you are using for installing k6 is being shutdown as a result of Bintray sunsetting its service on May 1st 2021.\e[0m
```
Comment (na--) [inline k6.spec]:
```suggestion
echo -e "\e[32mThe repo you are using for installing k6 is being shut down as a result of Bintray sunsetting its service on May 1st 2021.\e[0m
```

File: k6.spec [hunk 5]
```diff
@@ -9,17 +9,16 @@ Summary: Load testing for the 21st century
...
+The k6 team has made available a new repository where future packages will be published.
```
Comment (na--) [inline k6.spec]:
```suggestion
The k6 team has created a new package repository where future packages will be published.
```

**Review by na-- — APPROVED** [review]
LGTM besides the minor nitpicks

**Pull request merged**